_benchmark/metrics/metric-keys.md

```diff
@@ -0,0 +1,47 @@
+- `disk_io_write_bytes`: The number of bytes written to disk during the benchmark. On Linux, this metric corresponds to only the bytes that have been written by OpenSearch Benchmark. On Mac OS, it includes the number of bytes written by all processes.
+- `disk_io_read_bytes`: The number of bytes read from disk during the benchmark. The same caveats apply on Mac OS as for `disk_io_write_bytes`.
+- `node_startup_time`: The time in seconds it took from the process start until the node is up.
+- `node_total_young_gen_gc_time`: The total runtime of the young generation garbage collector across the whole cluster, as reported by the Nodes Stats API.
+- `node_total_young_gen_gc_count`: The total number of young generation garbage collections across the whole cluster, as reported by the Nodes Stats API.
+- `node_total_old_gen_gc_time`: The total runtime of the old generation garbage collector across the whole cluster, as reported by the Nodes Stats API.
+- `node_total_old_gen_gc_count`: The total number of old generation garbage collections across the whole cluster, as reported by the Nodes Stats API.
+- `node_total_zgc_cycles_gc_time`: The total time spent the Z Garbage Collector (ZGC) spent on garbage collecting across the whole cluster, as reported by the Nodes Stats API.
+- `node_total_zgc_cycles_gc_count`: The total number of garbage collections ZGC performed across the whole cluster, as reported by the Nodes Stats API.
+- `node_total_zgc_pauses_gc_time`: The total time ZGC spent in Stop-The-World pauses across the whole cluster, as reported by the Nodes Stats API.
+- `node_total_zgc_pauses_gc_count`: The total number of Stop-The-World pauses during ZGC execution across the whole cluster, as reported by the Nodes Stats API.
+- `segments_count`: The total number of open segments, as reported by the Index Stats API.
+- `segments_memory_in_bytes`: The total number of bytes used for all open segments, as reported by the Index Stats API.
+- `segments_doc_values_memory_in_bytes`: The number of bytes used for doc values, as reported by the Index Stats API.
+- `segments_stored_fields_memory_in_bytes`: The number of bytes used for stored fields, as reported by the Index Stats API.
+- `segments_terms_memory_in_bytes`: The number of bytes used for terms, as reported by the Index Stats API.
+- `segments_norms_memory_in_bytes`: The number of bytes used for norms as reported by the Index Stats API.
+- `segments_points_memory_in_bytes`: The number of bytes used for points as reported by the index stats API.
+- `merges_total_time`: The cumulative runtime of merges for primary shards, as reported by the index stats API. Note that this time is not wall clock time. If M merge threads ran for N minutes, Benchmark reports the amount of time as M * N minutes, not N minutes. These metrics records have an additional per-shard property that contains the times across primary shards in an array.
```

Comment:
In the second sentence, does "is not" mean "does not correspond to"? If so, apply to all affected list items.